setup.py

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env python
 from setuptools import setup
 
-version = "1.4.8"
-
 setup(
     name="django-widget-tweaks",
-    version=version,
+    use_scm_version={"version_scheme": "post-release"},
+    setup_requires=["setuptools_scm"],
     author="Mikhail Korobov",
     author_email="kmike84@gmail.com",
     url="https://github.com/jazzband/django-widget-tweaks",
     description="Tweak the form field rendering in templates, not in python-level form definitions.",
     long_description=open("README.rst").read() + "\n\n" + open("CHANGES.rst").read(),
+    long_description_content_type='text/x-rst',
     license="MIT license",
-    requires=["django (>=1.11)"],
+    requires=["django (>=2.0)"],
```

Comment:
Should this be >= 2.2 per support matrix?

Reply:
Hmm, yeah, fixed in eff492636b27ce4915d94e4991aac1451803a07c.